host-configs/llnl/blueos_3_ppc64le_ib_p9/clang@upstream_link_with_nvcc.cmake

```diff
@@ -35,38 +38,39 @@ set(MPI_HOME               "/usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-
 set(MPI_C_COMPILER         "${MPI_HOME}/bin/mpicc"   CACHE PATH "")
 set(MPI_CXX_COMPILER       "${MPI_HOME}/bin/mpicxx"  CACHE PATH "")
 
-set(MPIEXEC                "mpirun"  CACHE PATH "")
+set(MPIEXEC                "${MPI_HOME}/bin/mpirun"  CACHE PATH "")
 set(MPIEXEC_NUMPROC_FLAG   "-np"     CACHE PATH "")
 set(BLT_MPI_COMMAND_APPEND "mpibind" CACHE PATH "")
-
+set(BLT_MPI_LINK_FLAGS     "-Xlinker -rpath -Xlinker ${MPI_HOME}/lib" CACHE STRING "")
 
 #------------------------------------------------------------------------------
 # OpenMP support
 #------------------------------------------------------------------------------
 set(ENABLE_OPENMP ON CACHE BOOL "")
 
 # Override default link flags because linking with nvcc
-set(BLT_OPENMP_LINK_FLAGS "-Xlinker -rpath -Xlinker /usr/tce/packages/clang/${CLANG_VERSION}/ibm/omprtl/lib -L/usr/tce/packages/clang/${CLANG_VERSION}/ibm/omprtl/lib -lomp -lomptarget-nvptx" CACHE STRING "")
+set(OMP_HOME ${CLANG_HOME}/ibm/omprtl)
+set(BLT_OPENMP_LINK_FLAGS "-Xlinker -rpath -Xlinker ${OMP_HOME}/lib -L${OMP_HOME}/lib -lomp -lomptarget-nvptx" CACHE STRING "")
 
 
 #------------------------------------------------------------------------------
 # CUDA support
 #------------------------------------------------------------------------------
 set(ENABLE_CUDA ON CACHE BOOL "")
 
-set(CUDA_TOOLKIT_ROOT_DIR "/usr/tce/packages/cuda/cuda-9.2.88" CACHE PATH "")
+set(CUDA_TOOLKIT_ROOT_DIR "/usr/tce/packages/cuda/cuda-9.2.148" CACHE PATH "")
 set(CMAKE_CUDA_COMPILER "${CUDA_TOOLKIT_ROOT_DIR}/bin/nvcc" CACHE PATH "")
 set(CMAKE_CUDA_HOST_COMPILER ${MPI_CXX_COMPILER} CACHE PATH "")
 
-set (CUDA_ARCH "sm_60" CACHE PATH "")
-set (CMAKE_CUDA_FLAGS "-restrict -arch ${CUDA_ARCH} -std=c++11 --expt-extended-lambda -G" CACHE STRING "" )
+set (_cuda_arch "sm_60")
```

Comment:
@cyrush @mclarsen -- Re: #282 -- I changed the name here to highlight that this is not a cache variable, or related to internal workings of CMake or CUDA.